http/swagger.yml

```diff
@@ -8930,54 +8925,19 @@ components:
               type: boolean
             level:
               $ref: "#/components/schemas/CheckStatusLevel"
-    ThresholdBase:
+    Threshold:
       properties:
         level:
          $ref: "#/components/schemas/CheckStatusLevel"
         allValues:
           description: if true, only alert if all values meet threshold
           type: boolean
-    GreaterThreshold:
```

Comment:
Is this temporary? I thought that we had settled on the exiting types: https://github.com/influxdata/influxdb/issues/14581

Reply:
yeah, looks like it's just come kind of conflict, let me remove this part.

Reply:
gotcha, thanks!